notebooks/266-speculative-sampling/266-speculative-sampling.ipynb

```diff
@@ -23,8 +23,6 @@
    "id": "f97c435a",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        main_ov_model = OVModelForCausalLM.from_pretrained(main_model_id, device=current_device, from_transformers=True)
</pre><p>from_transformers flag is deprecated, could you please replace with export=True? Also as your code below compile models without optimum, this model classes instances used only for models conversion, right?</p><p> </p><p>I think maybe better to use optimum-cli for export models if they are not exists and them load model from directory like in code below? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>replaced with optimum-cli. Now looks better. Also replaced with 7b, now runs faster and less ram is used.</p>